README.md

```diff
@@ -1,17 +1,38 @@
-# Baremetal Deploy
+# OpenShift Baremetal Deploy
 
-Deployment artifacts for Bare Metal OpenShift KNI clusters
+This repository is intended to store resources and deployment artifacts
+for [bare metal](https://github.com/metal3-io/metal3-docs/blob/master/design/bare-metal-style-guide.md)
+OpenShift KNI clusters (currently OpenShift 4.3)
+
+It also contains optional features focused on low-latency workloads, NFV workloads, etc.
+
+## Installation artifacts
 
-## Current Artifacts
 * [Installation Steps](install-steps.md)
 * [Using Baremetal-prep.sh](baremetal-prep.md)
 
-## To Be Added
-* Additional step for disconnected install
-* Installation Troubleshooting
-* Installation scripts
-* Various feature yamls/manifest
+## Optional features
+
+* [Cluster Network Addons Operator](features/cnao/). This operator can be used to deploy additional
+networking components on top of Kubernetes/OpenShift cluster.
+* [CPU Manager](features/cpumanager/). CPU Manager manages groups of CPUs and constrains workloads
+to specific CPUs.
+* [Performance Tuning](features/performance). The manifests included in this folder enables
```

Comment:
via `Node Tuning` and `Machine config` operator

Reply:
Fixed, thanks